README.md

```diff
@@ -88,19 +92,37 @@ Tests in Civil's ecosystem require the use of [Ganache](https://github.com/truff
 yarn ganache
 ```
 
-some of the packages - most notably [`@joincivil/core`][core-url] depend on already deployed singletons. Our ganache is set-up to be deterministic and thus when you migrate contracts locally, singletons will have the same addresses as the artifacts.
+Finally run all the tests in the main repository:
 
 ```bash
-cd packages/contracts
-yarn migrate --network=ganache
+yarn test
 ```
 
-Finally run all the tests in the main repository:
+### Easy Setup with front-end
+
+We've added a helper command to assist you if you want to develop using the front-end and a local ganache. Just 3 simple commands.
+
+From the root directory:
 
 ```bash
-yarn test
+yarn ganache
 ```
 
+Once that is running, in another terminal, from the root directory:
+
```

Comment:
Should we add something about `contracts/config.json` being the place to add addresses for local ganache token distribution / appellate multi-sig?

Reply:
good point, i'll add that